pkg/config/config.go

```diff
@@ -855,3 +856,28 @@ func applyOverrides(config Config) {
 		config.Set(k, v)
 	}
 }
+
+// TracemallocEnabled is a helper to get the effective tracemalloc
+// configuration.
+func TracemallocEnabled(config Config) bool {
```

Comment:
I think we should unexpert both `TracemallocEnabled` and `GetNumWorkers` from the config package and call them both here https://github.com/DataDog/datadog-agent/blob/master/pkg/config/config.go#L627.

This function would edit the value of `tracemalloc_debug` (with a log line if enabled with python 2) instead of returning a value.

This would keep the configuration consistent with `configcheck` and `flare` commands.